ryantbrown/reaktion.js
# added ability to force responsive mode off

Useful when integrating reaktion.js menus into extant responsive menu systems. Might run a little counter to the aims of the project, I'm not sure. :)

Testing instructions:
- Verify that default reaktion.js behavior in the example is still okay.
- Set mobileOnly to true in the example.
- Observe that the menu is now rendered without the hamburger icon, with the root level fully opened.
